src/main/scala/no/ndla/learningpathapi/controller/InternController.scala

```diff
@@ -48,6 +46,14 @@ trait InternController {
       }
     }
 
+    get("/externalidtonewid/:external_id") {
```

Comment:
Er ikke kjempeglad i navnet på dette endepunktet. Det skal brukes for å populere taksonomien?
Hmm...hva med `get /id/:external_id` eller blir det bare tullete? Det leser så gærnt med ..get externalidtonewid..